config/locales/en.yml

```diff
@@ -46,8 +49,10 @@ en:
         other: "%{count} addresses found"
 
   # Custom error pages
+  invalid_reg_identifier_heading: Invalid registration number
```

Comment:
Think a typo. Assume it should be `invalid_reg_identifier_title`.

Reply:
Oops, yes! Will fix.